sale_payment_term_interest/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Sales Payment Term Interests
+============================
+
+This module allows to compute interest fees based on payment terms and
+to add a line in the sales orders with the computed amount.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * Install the `sale_payment_term_interest` module is the only things to do.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * Go to *Invoicing > Configuration > Payment Terms*
+ * On the lines of the payment terms, you can configure an interest
+   rate. It is based on the number of days.
```

Comment:
I agress with @gurnalex here. Interest is in most cases computed on an annual basis. But, dividing by 12 for monthly is not the way to do this, because interest is rarely compounded annually.